tasks/cli_tools.yml

```diff
@@ -0,0 +1,37 @@
+---
+
+# Compile available themes
+
+- name: "Update database"
+  command: "php symfony tools:upgrade-sql -B"
+  args:
+    chdir: "{{ atom_path }}/{{ atom_extra_path }}"
+  when: "atom_upgrade_sql|bool"
+
+- name: "Build nested set"
+  command: "php symfony propel:build-nested-set"
+  args:
+    chdir: "{{ atom_path }}/{{ atom_extra_path }}"
+  when: "atom_fix_data|bool"
+
+- name: "Generate slugs"
+  command: "php symfony propel:generate-slugs"
+  args:
+    chdir: "{{ atom_path }}/{{ atom_extra_path }}"
+  when: "atom_fix_data|bool"
+
+- name: "Clear cache"
+  command: "php symfony cache:clear"
+  args:
+    chdir: "{{ atom_path }}/{{ atom_extra_path }}"
+  notify:
+    - "Reload PHP service"
+
+# stdout is ignored, not very useful and too verbose (--quiet opt would be nice)
+- name: "Populate search index"
+  shell: "php symfony search:populate 1>/dev/null"
+  args:
+    chdir: "{{ atom_path }}/{{ atom_extra_path }}"
+  ignore_errors: "yes" # segfault in php7, not sure yet why!
+  environment: "{{ atom_pool_php_envs }}"
+  when: "atom_populate_index|bool"
```

Comment:
Yeah, more consistent than 'force_search_rebuild', that'll be another of those translated variables for the group_vars/ inventory file or the translation file, maybe.  :-)

Reply:
@DavidHume-ArtefactualSystems thanks! I forgot to copy it from our all playbook. I'll update the branch with this change ;)

Reply:
Branch updated!